mne_bids/__init__.py

```diff
@@ -3,10 +3,10 @@
 __version__ = '0.4.dev0'
 
 
-from mne_bids.write import (write_raw_bids, make_bids_folders, make_bids_basename,  # noqa: E501 F401
+from mne_bids.write import (write_raw_bids, make_bids_basename,  # noqa: E501 F401
                             make_dataset_description, write_anat,  # noqa: F401
                             get_anonymization_daysback,  # noqa: F401
                             _stamp_to_dt, _get_anonymization_daysback)
-
-from mne_bids.read import read_raw_bids, get_head_mri_trans  # noqa: F401
+from mne_bids.read import read_raw_bids, get_head_mri_trans, get_matched_empty_room  # noqa: F401
+from mne_bids.utils import make_bids_folders  # noqa: F401
 from mne_bids import commands  # noqa: F401
```

Comment:
all these should be relative imports

Reply:
this was discussed in https://github.com/mne-tools/mne-bids/pull/248 where we switched to absolute imports because of pep8